tests/validation_tests/asm.pnt

```diff
@@ -0,0 +1,67 @@
+// db <<<
+// 0, 69
+// 1, 42
+// >>>
+
+storage {
+    s1: int,
+    s2: int,
+}
+
+predicate foo() {
+    let y = 5555;
+    let z = 3333;
+    let w = 2222;
+
+    let sum: int = asm(w, y, z) {
+        PUSH 0
+        LOD // load w
+        PUSH 1
+        LOD // load y
+        ADD
+        PUSH 2
+        LOD // load z
+        ADD
+    };
```

Comment:
Shouldn't `w`, `y` and `z` have prefix sizes?  I thought `w`s length would be at `0`, and it would be 1.  So `w` at 1, `y` at 3 and `z` at 5?

Reply:
I've been informed that the prefix sizes have been removed, since they're pretty much redundant.

Reply:
I'll still open an issue to discuss whether sizes are needed or not. They may become relevant when dynamic arrays are added.